### PR TITLE
lc102, lc637

### PR DESCRIPTION
**Pull request by nicwoj**

### CODE REVIEW

File: nicoleWojewoda/assignments/treesBFS/lc102/lc102.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent its level-by-level traversal. You should populate the values of all nodes of each level from left to right in separate sub-arrays.
```
Comment (MelissaPastore) [inline nicoleWojewoda/assignments/treesBFS/lc102/lc102.js:3]: Nice overview here. Would also suggest writing some additional test cases, especially for an edge case.

File: nicoleWojewoda/assignments/treesBFS/lc637/lc637.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent the averages of all of its levels.
+
+/*
+input: the root node of a binary tree
```
Comment (MelissaPastore) [inline nicoleWojewoda/assignments/treesBFS/lc637/lc637.js:6]: Nice overview here. Would also suggest writing some additional test cases and also analyzing the time and space complexity of the problem.

**Review by MelissaPastore — APPROVED** [review]
